packages/business-logic-plugin/BusinessLogicBase.ts

```diff
@@ -1,72 +0,0 @@
-/*
```

Comment:
With this change, files such as business-logic-plugin are deleted. I will ask you to modify the PR so that it does not reflect the file change without deleting the file.